server/hotrod/src/test/java/org/infinispan/server/hotrod/HotRodMergeTest.java

```diff
@@ -27,7 +27,7 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test(groups = "functional", testName = "server.hotrod.HotRodMergeTest")
+@Test(groups = "functional", testName = "server.hotrod.HotRodMergeTest", enabled = false)
```

Comment:
I wonder should we just add it to the unstable group instead? I am not up to speed on how we are disabling nowadays :)

Reply:
Yeah, we normally add it to the `unstable` group, e.g. `groups = {"functional", "unstable"}`

Reply:
I didn't realise, good to know. I'll change it